src/frontend/komponenter/Felleskomponenter/MånedÅrInput/MånedÅrVelger.tsx

```diff
@@ -71,14 +62,13 @@ const MånedÅrVelger: React.FC<Props> = ({
     }, [value]);
 
     return (
-        <div className={className} style={lesevisning ? { minWidth: '140px' } : {}}>
-            {label && <DatolabelStyle htmlFor={id}>{label}</DatolabelStyle>}
```

Comment:
Flytter label inn i Måned-komponenten for å få med låsen fra aksel i lesevisning